repair/row.hh

```diff
@@ -69,7 +69,17 @@ public:
         if (!_fm) {
             throw std::runtime_error("empty size due to empty frozen_mutation_fragment");
         }
-        return _fm->representation().size();
+        auto size = _fm->representation().size();
+        if (_boundary) {
+            size += _boundary->pk.external_memory_usage() + _boundary->position.external_memory_usage();
+        }
+        if (_mf) {
+            size += _mf->memory_usage();
+        }
```

Comment:
@asias why do we have both frozen and non-frozen mutations here? isn't it redundant?

Reply:
The mf is used to avoid unfreezing from the frozen mutation to mf again when applying it. The frozen mutation is kept because the repair master could send the frozen mutation to other peer nodes again.

Reply:
Aha. Maybe we should prefer the CPU hit to the memory hit.

Reply:
It is an option to de serialize it again when we need it.  But we use the mf almost immediate, it is too wasteful to do it again.